sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/inputFileBlock.scala

```diff
@@ -42,8 +43,8 @@ case class InputFileName() extends LeafExpression with Nondeterministic {
 
   override def doGenCode(ctx: CodegenContext, ev: ExprCode): ExprCode = {
     val className = InputFileBlockHolder.getClass.getName.stripSuffix("$")
-    ev.copy(code = s"final ${CodeGenerator.javaType(dataType)} ${ev.value} = " +
-      s"$className.getInputFilePath();", isNull = FalseLiteral)
+    ev.copy(code = code"final ${CodeGenerator.javaType(dataType)} ${ev.value} = " +
+      code"$className.getInputFilePath();", isNull = FalseLiteral)
```

Comment:
the `Blocks.code` is defined as `override def code: String = blocks.map(_.toString).mkString("\n")`, is it OK here?

maybe we can do
```
val typeDef = s"final ${CodeGenerator.javaType(dataType)}"
ev.code(code = code"$typeDef ${ev.value} = $className.getInputFilePath();", ...)
```